oeps/oep-0037-bp-test-data.rst

```diff
@@ -94,60 +94,55 @@ The function can take one or more positional arguments if useful for using it in
+    ./manage.py load_dev_data /tmp/example2.yaml
 
-.. code-block:: bash
+To keeping data loading modular, dev data specification for each IDA should live in its own yaml file.
 
-    ./manage.py load_test_data test_data.from_dev_data_v1 path/to/file.yaml
-    make load_test_data DATA_FUNCTION=test_data.from_dev_data_v1 DATA_FUNCTION_ARGS=http://domain.org/path/to/file.yaml
+Each ``load_dev_data`` management command should take the specification from the yaml file and call on the data generation fuctions that correspond to keys in yaml file.
 
 Each data loading function should be executed during the respective IDA's test suite, in order to ensure that it stays functional across schema and code changes.  This also makes it clear what change triggered failure to load the data, making it much faster to make the appropriate fixes.
 
-Because the test data doesn't contain primary keys, loading the same data a second time into an environment will usually result in a second copy of the test data being created in it.  If there is desire for a set of test data that can be updated or reset in an environment in which it already exists, its data loading functions should be deliberately written to be capable of either deleting or updating any data they previously loaded (via known field value lookups, etc.)
+Because the dev data doesn't contain primary keys, loading the same data a second time into an environment will usually result in a second copy of the dev data being created in it.  If there is desire for a set of dev data that can be updated or reset in an environment in which it already exists, its data loading functions should be deliberately written to be capable of either deleting or updating any data they previously loaded (via known field value lookups, etc.)
 
 Data Files
 ----------
 
-When creating a more complex data set, especially one containing data to be loaded into multiple Open edX services, it is often best to describe the data in a YAML file.  The path or URL of this file can then be passed as an argument to each data loading function which uses the information in it to create database records for a particular service as shown above.  Requirements for such data sets vary widely, so we won't attempt to enforce a single standard schema for them, but each such file should declare the name of the schema it has chosen to use.  Such a file might look like this:
+Devstack dev data will be specified in a YAML file.  The path or URL of this file is passed to the ``load_dev_data`` management command, which uses the information in it to call the appropriate data generation function to create database records for a particular service as shown above. Such a file might look like this:
```

Comment:
I think this paragraph is a bit vague. What is 'devstack dev data' in this case? Is it specifying for one IDA? Multiple IDAs? I think you need to clarify what is going into the YAML files.

Reply:
I forgot to change this "Data Files' section to be a subsection under "Loading Data" section. Most of the info you asked for in this question is written in the paragraphs above.

In my next commit, I will:
- change "Data Files" to be a subheading under "Loading Data"
- Clarify that the yaml file is for a single IDA.

Reply:
Changes made, please review again.

Reply:
Feel free to let me know if it is still too vague after taking into considering the above.